reference/TubePlayer/src/TubePlayer/TubePlayer.csproj

```diff
@@ -38,6 +38,8 @@
       Navigation;
       Http;
       MediaElement;
+      SkiaRenderer;
+      HttpRefit;
```

Comment:
We can remove the Http feature since the relevant dependencies for it would be pulled in by the HttpRefit feature